emails/tests/utils_tests.py

```diff
@@ -67,6 +73,21 @@ def test_generate_relay_From_with_linebreak_chars(self):
         )
         assert formatted_from_address == expected_formatted_from
 
+    def test_generate_relay_From_with_premium_user(self):
+        premium_user = make_premium_test_user()
+        premium_profile = premium_user.profile_set.first()
+        original_from_address = '"foo bar" <foo@bar.com>'
+        formatted_from_address = generate_relay_From(original_from_address, premium_profile)
+
+        expected_encoded_display_name = (
+            '=?utf-8?b?IiJmb28gYmFyIiA8Zm9vQGJhci5jb20+IFt2aWEgUmVsYXldIg==?='
+        )
+        expected_formatted_from = '%s %s' % (
+            expected_encoded_display_name, '<%s>' % self.premium_from
+        )
```

Comment:
Nit: Two-levels of formatting is weird....

```suggestion
        expected_formatted_from = '%s <%s>' % (
            expected_encoded_display_name, self.premium_from
        )
```

An f-string may work, or may look worse.